packages/api-core/src/api.js

```diff
@@ -125,7 +126,7 @@ export default class AvApi {
       location = response.headers.location || response.headers.Location;
     }
 
-    return location;
+    return resolveUrl(location);
```

Comment:
Per the spec, it should be relative to the request URL, not the window. `resolveUrl` doesn't know the request URL, and only uses the window's url.

Maybe `resolveUrl` should optionally accept a second parameter which is the base url. Then you can pass the request URL along with the location to derive the fully qualified path (needed if the request is to a different domain).

Reply:
that sounds good.

Reply:
@TheSharpieOne https://github.com/Availity/sdk-js/pull/102/files#diff-0fb53fc9a397f9b2885999768db1c330R133